<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Warzone Model Post-production Program</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="WZMOpenGLWidget" name="glView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport_3DS">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_3DS"/>
    <addaction name="actionImport_OBJ"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAnimation"/>
    <addaction name="actionCulling"/>
    <addaction name="actionWireframe"/>
    <addaction name="actionAxis"/>
    <addaction name="separator"/>
    <addaction name="actionFramesEditor"/>
    <addaction name="actionConnectorsEditor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpContents"/>
    <addaction name="separator"/>
    <addaction name="actionAboutApplication"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuConversion">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSwapYZ"/>
    <addaction name="actionReverseWinding"/>
    <addaction name="actionFlipVerticalTexCoords"/>
    <addaction name="actionScaleModel"/>
   </widget>
   <addaction name="menuImport_3DS"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuConversion"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="animationDockWidget">
   <property name="windowTitle">
    <string>Animation Frames</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="animationDockWidgetContents">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="animationTableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="animationlLayout">
       <item>
        <widget class="QPushButton" name="prependFrameButton">
         <property name="text">
          <string>Prepend</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="appendFrameButton">
         <property name="text">
          <string>Append</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeFrameButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="animationSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="connectorDockWidget">
   <property name="windowTitle">
    <string>Connectors</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="connectorDockWidgetContents">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="connectorTableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="connectorlLayout">
       <item>
        <widget class="QPushButton" name="prependConnectorButton">
         <property name="text">
          <string>Prepend</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="appendConnectorButton">
         <property name="text">
          <string>Append</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeConnectorButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="connectorSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionImport_3DS">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import 3DS...</string>
   </property>
  </action>
  <action name="actionImport_OBJ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import OBJ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Animate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCulling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Culling</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show axis</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSwapYZ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Swap Y and Z axis</string>
   </property>
  </action>
  <action name="actionReverseWinding">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reverse polygon winding</string>
   </property>
  </action>
  <action name="actionFlipVerticalTexCoords">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Flip vertical texcoords</string>
   </property>
  </action>
  <action name="actionScaleModel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scale model</string>
   </property>
  </action>
  <action name="actionEditFrames">
   <property name="text">
    <string>Frames</string>
   </property>
  </action>
  <action name="actionEditConnectors">
   <property name="text">
    <string>Connectors</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionAboutApplication">
   <property name="text">
    <string>About QWZM...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionFramesEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Frames Editor</string>
   </property>
  </action>
  <action name="actionConnectorsEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connectors Editor</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WZMOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>wzmglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
